doc/data/messages/o/order-dependent-resolution/bad.py

```diff
@@ -0,0 +1,14 @@
+class HarshParent:
+    @staticmethod
+    def scold():
+        return "Don't do that!"
```

Comment:
```suggestion
class EmotionalBlackmailingParent:
    @staticmethod
    def scold():
        return "If you do that, mommy won't love you anymore !"
```

This is not a real suggestion. Great example, the possibility are endless !! 😄